tools/integration-tests/tester/framework/docker.go

```diff
@@ -144,6 +145,7 @@ func (d *DockerContainer) CreateGoShimmerPeer(config GoShimmerConfig) error {
 			fmt.Sprintf("--mana.allowedConsensusPledge=%s", func() string {
 				return strings.Join(config.ManaAllowedConsensusPledge[:], ",")
 			}()),
+			fmt.Sprintf("--database.forceCacheTime=%d", config.ForceCacheTime),
```

Comment:
Since now `database.forceCacheTime` expects a `time.Duration` type which looks like `10s` instead of just an int you should use "%s" instead of "%d"
```suggestion
			fmt.Sprintf("--database.forceCacheTime=%s", config.ForceCacheTime),
```